File: RedSheet.DbEntities/Models/vMeetingRequirement.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vMeetingRequirements", Schema = "dbo")]
	[Serializable]
	public partial class vMeetingRequirement  
	{
 
	[DatabaseGenerated(DatabaseGeneratedOption.None)] 
	[Key]
	public int CultureCountryId { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int MeetingRequirementId { get; set; } 
 
	[Required] 
	[MaxLength(500)]
	public string MeetingRequirementValue { get; set; } 

	public vMeetingRequirement()
	{
    }

    }
}

File: redsheet-portal/src/app/database-models/v-meeting-requirement.ts
import { required, maxLength, range } from '@rx/annotations';

export class vMeetingRequirement {
    constructor(vMeetingRequirement?: vMeetingRequirement )  {
        let properties = [ "cultureCountryId", "meetingRequirementId", "meetingRequirementValue",];
        for (let property of properties)
            if (vMeetingRequirement && vMeetingRequirement[property])
                this[property] = vMeetingRequirement[property];
    }
 
	cultureCountryId : number =   0 ;
 
    @range(1,2147483647)
	meetingRequirementId : number =   undefined;
 
    @required()
    @maxLength(500)
	meetingRequirementValue : string =   undefined;
}
